### PR TITLE
Update the amp-accordion documentation

### PR DESCRIPTION
**Pull request by powdercloud**
Update the amp-accordion documentation to reflect the implemented validation logic. Slightly more strict / simpler.

### CODE REVIEW

File: extensions/amp-accordion/amp-accordion.md [hunk 1]
```diff
@@ -40,11 +40,9 @@ limitations under the License.
 Each of the `amp-accordion` component’s immediate children is considered a section in the accordion. Each of these nodes must be a `<section>` tag.
 
 - An `amp-accordion` can contain one or more `<section>`s as its direct children.
-- Each `<section>` must contain only two direct children.
-- The first child (of the section) will be considered as the heading of the section. Clicking/tapping on this section will trigger the expand/collapse behaviour.
-- The second child (of the section) will be the content or the section
-- There is no restriction on the type of tags that could be used for the `<section>`’s children.
-- Any additional children of the `<section>` would be ignored not be displayed. (This should just be a safety backup and should be enforced in the validator)
+- Each `<section>` must contain exactly two direct children.
+- The first child (of the section) must be one of `h1`, `h2`, ..., `h6` and is the heading of the section.
```
Comment (sriramkrish85) [inline extensions/amp-accordion/amp-accordion.md]: we should also support <header> as pointed out by @cramforce on the design doc.(not required in this PR)

File: extensions/amp-accordion/amp-accordion.md [hunk 2]
```diff
@@ -40,11 +40,9 @@ limitations under the License.
 Each of the `amp-accordion` component’s immediate children is considered a section in the accordion. Each of these nodes must be a `<section>` tag.
```
Comment (sriramkrish85) [inline extensions/amp-accordion/amp-accordion.md]: above this 
```html
   <tr>
     <td class="col-fourty"><strong>Availability</strong></td>
     <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a>; no validations yet.</td>
   </tr>
```
should be replaced with 
```html
  <tr>
    <td width="40%"><strong>Availability</strong></td>
    <td>Stable</td>
  </tr>
```

File: extensions/amp-accordion/amp-accordion.md (hunk 1 above)
Comment (powdercloud) [inline extensions/amp-accordion/amp-accordion.md]: Did github's rendering eat part of your comment here? What else should we support? Sorry that I missed something.

Comment (sriramkrish85) [inline extensions/amp-accordion/amp-accordion.md]: `header` tag

**Comment by sriramkrish85** [comment]
Ref: https://github.com/ampproject/amphtml/issues/2061

File: extensions/amp-accordion/amp-accordion.md (hunk 1 above)
Comment (powdercloud) [inline extensions/amp-accordion/amp-accordion.md]: Ah cool. I'll do that, and once it's out we can update this doc again.

File: extensions/amp-accordion/amp-accordion.md (hunk 2 above)
Comment (powdercloud) [inline extensions/amp-accordion/amp-accordion.md]: done

File: extensions/amp-accordion/amp-accordion.md [hunk 3]
```diff
@@ -22,8 +22,8 @@ limitations under the License.
     <td>An accordion provides a way for viewers to glance at the content outline and jump to any section. This is helpful for handheld mobile devices where even a couple of sentences into a section requires scrolling.</td>
   </tr>
   <tr>
-    <td class="col-fourty"><strong>Availability</strong></td>
-    <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a>; no validations yet.</td>
+    <td width="40%"><strong>Availability</strong></td>
```
Comment (sriramkrish85) [inline extensions/amp-accordion/amp-accordion.md]: sorry it should be - class="col-fourty":)

Comment (powdercloud) [inline extensions/amp-accordion/amp-accordion.md]: done

**Comment by sriramkrish85** [comment]
LGTM

**Pull request merged**